test/functional/change_stream.test.js

```diff
@@ -2850,3 +2858,99 @@ describe('Change Streams', function() {
     });
   });
 });
+
+describe('Change Stream Resume Error Tests', function() {
+  function withChangeStream(testName, callback) {
+    return function(done) {
+      const configuration = this.configuration;
+      const client = configuration.newClient();
+      client.connect(err => {
+        expect(err).to.not.exist;
+        const db = client.db('changeStreamResumErrorTest');
+        db.createCollection(testName, (err, collection) => {
+          expect(err).to.not.exist;
+          const changeStream = collection.watch();
+          callback(collection, changeStream, () =>
+            changeStream.close(() => collection.drop(() => client.close(done)))
+          );
+        });
+      });
+    };
+  }
+  it('(events) should continue iterating after a resumable error', {
```

Comment:
`(events)` implies a `context` to me

Reply:
I cleaned up the descriptions on those tests.